book/src/csi-proxy.md

```diff
@@ -0,0 +1,44 @@
+# CSI Proxy
+
+## Status and Releases
+
+**Git Repository:** [https://github.com/kubernetes-csi/csi-proxy](https://github.com/kubernetes-csi/csi-proxy)
+
+**Status:** Beta starting with v0.2.0
+
+Status | Min K8s Version | Max K8s Version
+--|--|--
+Alpha | 1.18 | - 
+Beta | 1.18  | -
+
+## Description
+
+CSI Proxy is a binary that exposes a set of gRPC APIs around storage operations over named pipes in Windows. A container, such as CSI node plugins, can mount the named pipes depending on operations it wants to exercise on the host and invoke the APIs.
+
+Each named pipe will support a specific version of an API (e.g. v1alpha1, v2beta1) that targets a specific area of storage (e.g. disk, volume, file, SMB, iSCSI).
+
+## Usage
+
+Run csi-proxy.exe binary directly on Window node. Command line options
+
+* `-kubelet-csi-plugins-path`: This is the prefix path of the Kubelet plugin directory in the host file system (`C:\var\lib\kubelet` is used by default).
+
+* `-kubelet-pod-path`: This is the prefix path of the kubelet pod directory in the host file system (`C:\var\lib\kubelet` is used by default).
```

Comment:
we can call it kubelet-path

Reply:
Can we change this for the GA v1 version? I think this field is a little confusing because pod path is usually "/var/lib/kubelet/pods". Especially because this path and the csi-plugins-path is the same. Can we combine the two options?

Reply:
yes, we plan to do it.